Multisig/UI/Transactions/TransactionDetails/TransactionDetailsViewModel.swift

```diff
@@ -86,4 +86,26 @@ class TransactionDetailsViewModel: BasicLoadableViewModel {
             return try App.shared.clientGatewayService.transactionDetails(id: id!)
         }
     }
+
+    func sign(safeAddress: Address) {
+        guard let transferTx = transactionDetails as? TransferTransactionViewModel,
+              let transaction = transferTx.transaction else { return }
+        Just(safeAddress)
+            .receive(on: DispatchQueue.global())
+            .tryMap { _ in
```

Comment:
I think it's better here to use `address` and use it in the call below, so that we don't need to capture anything from the outer method (dk why I say this but it feels better to me)

Reply:
ok